src/mappings/util/metadata.ts

```diff
@@ -36,7 +36,11 @@ export async function fetchMetadata(url: string, job: Queue.Job) {
     try {
         const { status, data } = await api.get(finalUrl)
         if (status < 400) {
-            return data
+            if (data && typeof data === 'object' && !Array.isArray(data)) {
+                return data
+            }
+            job.log(`Invalid response from ${url}`)
+            job.log(data)
```

Comment:
Does this log correctly, given `data` appears to be parsed?

Reply:
flow only reach here if the `if(typeof data === 'object')` fails